test/infrastructure/kind/mapper_test.go

```diff
@@ -44,10 +44,18 @@ func TestGetMapping(t *testing.T) {
 			k8sVersion:  semver.MustParse("1.23.17"),
 			customImage: "foo",
 			expectedMapping: Mapping{
-				Mode:  Mode0_20,
+				Mode:  Mode0_21,
```

Comment:
this change I am not entirely confident in based on the test name `No match for custom image fallback on K8s version match`